appium/tests/specs/live/composeEmail/CheckRecipientPopup.spec.ts

```diff
@@ -0,0 +1,29 @@
+import {MailFolderScreen, NewMessageScreen, SetupKeyScreen, SplashScreen} from '../../../screenobjects/all-screens';
+
+import {CommonData} from '../../../data';
+
+describe('COMPOSE EMAIL: ', () => {
+
+  it('check added recipient popup', async () => {
+
+    const recipient1 = CommonData.emailWithMultipleRecipientsWithCC.sender;
+    const recipient2 = CommonData.recipientWithExpiredPublicKey.email;
+    const recipient3 = CommonData.recipientWithRevokedPublicKey.email;
+
+    await SplashScreen.login();
+    await SetupKeyScreen.setPassPhrase();
+    await MailFolderScreen.checkInboxScreen();
+
+    await MailFolderScreen.clickCreateEmail();
+
+    await NewMessageScreen.setAddRecipient(recipient1);
+    await NewMessageScreen.setAddRecipient(recipient2);
+    await NewMessageScreen.setAddRecipient(recipient3);
+
+    await NewMessageScreen.checkCopyForAddedRecipient(recipient1, 0);
+
+    await NewMessageScreen.deleteAddedRecipient(2);
+    await NewMessageScreen.deleteAddedRecipient(1);
+    await NewMessageScreen.deleteAddedRecipient(0);
```

Comment:
In all of the popups, we should also check it displays expected email and name. Can be in another issue.